packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2922,19 +2922,15 @@ export class ContainerRuntime
 
 		assert(this.outbox.isEmpty, 0x3d1 /* Can't trigger summary in the middle of a batch */);
 
+		// We close the summarizer and download a new snapshot and reload the container
 		let latestSnapshotVersionId: string | undefined;
-		if (refreshLatestAck) {
-			const latestSnapshotInfo = await this.refreshLatestSummaryAckFromServer(
+		if (refreshLatestAck === true) {
+			return this.closeSummarizerOnSummaryStateStale(
```

Comment:
Maybe first ovewrite the `error` on the result object to better describe this case?

Reply:
Ok looks like this is the only callsite.  Then can we call this function and the error it returns something else?  Something like `prefetchLatestSummaryThenClose(...)` and `error: "Unsupported option 'refreshLatestAck'"`